src/views/emails/template-parts/body/order/order-gateway-data.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Event Tickets Emails: Order Total
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/emails/template-parts/body/order/order-gateway-data.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version 5.5.11
+ *
+ * @since 5.5.11
+ *
+ * @var Tribe__Template                    $this               Current template object.
+ * @var \TEC\Tickets\Emails\Email_Abstract $email              The email object.
+ * @var string                             $heading            The email heading.
+ * @var string                             $title              The email title.
+ * @var bool                               $preview            Whether the email is in preview mode or not.
+ * @var string                             $additional_content The email additional content.
+ * @var bool                               $is_tec_active      Whether `The Events Calendar` is active or not.
+ * @var \WP_Post                           $order              The order object.
+ */
+
+if ( empty( $order )  ) {
+	return;
+}
+
+if ( empty( $order->gateway_order_id )  ) {
+	return;
+}
+
+$gateway_order_id_string = sprintf(
+	// Translators: %s - The order gateway ID.
+	__( 'Gateway Order #%s', 'event-tickets' ),
+	$order->gateway_order_id
+);
```

Comment:
I suggest that we add the Gateway dashboard link as well instead of showing just the ID. With that the site admins won't need to check the orders page again to find details for that transaction.

https://github.com/the-events-calendar/event-tickets/blob/1e2f0ff853003078e8c87364b4f370fddb060030/src/Tickets/Commerce/Gateways/Contracts/Gateway_Interface.php#L137

Reply:
We can't since the receipt is not an email to the admin, it's to the customer.

Reply:
Thank you!